### ISSUE TITLE
It would be helpful to see examples in the context of other content on the page

### ISSUE THREAD

**Issue by mbifulco** [wwebdev/separator-generator#9]
Fist off - thanks for creating this repo! It's simple, fantastic, and useful.  That's a heck of an accomplishment.

I'd love to see a small change to the site - it would be nice to see the individual separator examples placed in the context of other content on the page.  As they are presented now, the examples don't really point out where titles/text/images/etc should be placed to use the separators.

Basically, a few scenarios:

- separator as **top** of page - people might want a nice separator at the top of a squeeze page, for example, for aesthetic purposes
- separator as **middle** of page - how do I use the separator to separate content? 
- separator as **bottom** of page - how do I use the separator as the footer of my site? 

Thanks again 🙌

**Comment by Vincenius** [comment]
Hey man, thanks for the feedback. That's a good idea putting it into context. I'll think of a way how to approach that - Let's see when I have time to actually do this :)
